### DISCUSSION TITLE
the Z-axis direction of the custom 3D sphere model simulation error

### DISCUSSION THREAD

**Discussion by wdz00** [JuliaHealth/KomaMRI.jl#521] [Q&A]
Hi，
I created a 3D spherical body phantom（T1=1000ms T2=T2S=80ms） and used pulseq to create a 3D sequence (nodes，use TR=500ms to spoil Mxy). When the set FOV[80mm] in Z is smaller than the part of the sphere body model where ρ > 0[-78<z<43mm], the reconstruction result looks fine. However, when the set FOV(200mm) in Z exceeds the part of the sphere body model where ρ > 0, there seems to be a large amount of signal appearing in the regions where there should be no signal. I'm not sure why the results are different when the FOVZ exceeds the body model size in the Z direction compared to the X and Y directions.
FOV=[X Y Z,100 100 80mm]   [NX NY NZ]=[64 64 10]
![image](https://github.com/user-attachments/assets/c579cc45-be8e-497e-830a-935aede07779)
FOV=[X Y Z,100 100 200mm]   [NX NY NZ]=[64 64 10]
![image](https://github.com/user-attachments/assets/792de32d-f2b0-47bd-b84c-4465b70af9cd)
To make the difference much clearer, I created another sphere phantom with more spins and applied a larger FOV (80 mm) in the Z direction, which exceeds the region of the sphere body model where ρ > 0 (-12<z<12mm). Here are the results:
![image](https://github.com/user-attachments/assets/8509d5aa-bf19-4460-bb24-609012376254)
Using a smaller FOV (20 mm) improves the results, but some errors still remain in the last three images.
![image](https://github.com/user-attachments/assets/2a40c2e0-a5ad-4992-bc6b-987b2beac249)

**Comment by wdz00** [comment]
Here I upload my sequence and phantom creat files
[GRE_TEST.zip](https://github.com/user-attachments/files/18072096/GRE_TEST.zip)

**Reply by cncastillo** [reply]
Hi! Do you have the script that is doing the simulation? 

It is not completely clear to me what FOV you are reducing, FOVz of the sequence or the phantom? Isn't the problem just aliasing in the z-dimension?

btw, you can write a phantom to a file by using the `write_phantom`  function.

**Reply by wdz00** [reply]
Thank you for your suggestion. I will try the write_phantom function and use the Julia script for the simulation.I am just getting familiar with Julia, so I have only used KomaUI for simplicity. I apologize if my explanation was unclear. I used two phase encodings to encode the Y and Z directions. By FOVZ, I mean the distance in the Z direction covered by the phase encoding in my sequence. The issue isn't related to reducing the FOV; on the contrary, when I extend the phase encoding range in the Z direction beyond the region where the phantom's density ρ > 0, signals appear in the region where ρ = 0, similar to the first 7 and last 7 images in the third figure. I’ve also created a simple diagram to help explain Fig3.
![image](https://github.com/user-attachments/assets/41f5ca73-75f2-4e3a-94f1-f95df424720a)

**Reply by cncastillo** [reply]
Thanks for the explanation, the issue is much clearer now. I will take a look at the code during the weekend :).

Using the UI is fine, hopefully this is not a problem with the UI.

**Reply by cncastillo** [reply]
BTW are all the images plotted in the same scale? Sometimes the values are very small but you're still able to see something. Just want to discard this being a "plotting" problem.
